modules/client/src/main/scala/lucuma/itc/client/ItcClient.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc.client
+
+import cats.Applicative
+import cats.effect.Async
+import cats.effect.Ref
+import cats.effect.Resource
+import cats.syntax.applicativeError.*
+import cats.syntax.apply.*
+import cats.syntax.either.*
+import cats.syntax.flatMap.*
+import cats.syntax.functor.*
+import clue.TransactionalClient
+import clue.http4s.Http4sBackend
+import io.circe.syntax.*
+import lucuma.core.model.Observation
+import org.http4s.Uri
+import org.http4s.jdkhttpclient.JdkHttpClient
+import org.typelevel.log4cats.Logger
+
+/**
+ * Client for calling the ITC on the JVM.
+ */
+trait ItcClient[F[_]] {
+
+  // TODO: chart, version, etc.
+
+  def spectroscopy(
+    input:    SpectroscopyModeInput,
+    useCache: Boolean = true
+  ): F[Either[Throwable, List[SpectroscopyResult]]]
+
+}
+
+object ItcClient {
+
+  def create[F[_]: Async: Logger](
+    uri: Uri
```

Comment:
Would it make sense to make this generic and take a `Client` instance? This way it could be cross built to js

Reply:
@cquiroz  That seems like a good idea to me.  I added a `Client` parameter.  How does that look?

Reply:
Looks good 👍